WNPRC_EHR/src/org/labkey/wnprc_ehr/notification/NotificationToolkit.java

```diff
@@ -1035,7 +1189,123 @@ public String setColumnTextColor(int columnToColor, String textColor) {
             );
             return returnStyle.toString();
         }
+
+        //TODO: Add documentation.
+        public String setHeaderRowBackgroundColor(String headerColor) {
+            StringBuilder returnStyle = new StringBuilder();
+            returnStyle.append(
+                    "th:nth-child(n) {background: " + headerColor + "}"
+            );
+            return returnStyle.toString();
+        }
+
+//        public String setRowBackgroundColor(Integer[] rowsToHighlight, String highlightColor) {
+//
+//        }
+    }
+
+    static class NotificationRevampTable
```

Comment:
getTableAsHTML was my original function a few months ago when I started working on this.  The new NotificationRevampTable object is a much better way to create an HTML table because it uses a constructor with optional parameters.  As an object, it'll be much easier to work with and expand on for future notifications.  I'm also planning to remove the original getTableAsHTML function and use this new object in place of it.